demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -220,6 +221,97 @@ def generate_setup_section(yaml_data: dict):
     return section
 
 
+# Incident Mirroring
+
+def is_configuration_exists(yml_data: dict, display: str):
+    """
+    Args:
+        yml_data: yml data of the integration
+        display: configuration to search
+
+    Returns:
+        the cong if display exists in the yml_data, else none.
+
+    """
+
+    for conf in yml_data.get('configuration', []):
+        if conf.get('display') == display:
+            return conf
+    return None
+
+
+def generate_mirroring_section(yaml_data: dict):
+    """
+
+    Args:
+        yaml_data: dict representing the yml file of the integration.
+
+    Returns: markdown section of Incident Mirroring.
+
+    """
+    integration_name = format(yaml_data['display'])
+    directions = {
+        'None': 'Turns off incident mirroring.',
+        'Incoming': 'Any changes in CrowdStrike Falcon incidents (state, status, tactics, techniques, objectives, '
+                    'tags, hosts.hostname) will be reflected in XSOAR incidents.',
+        'Outgoing': 'Any changes in XSOAR incidents will be reflected in CrowdStrike Falcon incidents (tags, status).',
+        'Incoming And Outgoing': 'Changes in XSOAR incidents and CrowdStrike Falcon incidents will be reflected in both directions.'
+    }
+
+    section = [
+        '## Incident Mirroring',
+        '',
+        f'You can enable incident mirroring between Cortex XSOAR incidents and {integration_name} incidents (available from Cortex XSOAR version 6.0.0).',
+        'To setup the mirroring follow these instructions:',
+        '1. Navigate to **Settings** > **Integrations** > **Servers & Services**.',
+        f'2. Search for {integration_name} and select your integration instance.',
+        '3. Enable Fetches incidents.'
+    ]
+
+    index = 4
+    if is_configuration_exists(yaml_data, 'Incidents fetch query'):
+        section.append(
+            f'{index}. Optional: You can go to the Incidents fetch query parameter and select the query to fetch the incidents from {integration_name}.')
+        index = index + 1
+    if is_configuration_exists(yaml_data, 'Mirroring tag'):
+        section.append(f'{index}. Optional: You can go to the Mirroring tag parameter and select the tags used to '
+                       f'mark incident entries to be mirrored.')
+        index = index + 1
```

Comment:
The configuration is not called `Mirroring tag`, but there are 3 mirroring tags that have 3 configuration that can be added (none of them, all of them, or each other combination)- `comment_tag`, `work_notes_tag` and `file_tag`.
Maybe this was misunderstood as **the incident** might hold `mirror_tags` incident field.